### DISCUSSION TITLE
Integrate with superjson

### DISCUSSION THREAD

**Discussion by ksandinunikum** [gqty-dev/gqty#1616] [Ideas]
Would it be possible to integrate with [superjson](https://github.com/blitz-js/superjson)? It would be great to build in enhanced encodings into the generated client, so that the consumer of the client doesn't have to manually parse things like Date /RegExp objects (or whatever else one might need). Personally I'd like to use it for Date objects.

**Comment by vicary** [comment]
You are really free to edit the client file after codegen, e.g. `npx @gqty/cli@canary`.

1. Make sure your server returns superjson.
2. Replace the line `return JSON.parse(text);` with `return superjson.parse(text);` in your GQty client.

**Comment by ksandinunikum** [comment]
That's great, thanks!
